FitGoal/FitGoal/Singup&Singin/SocialMediaAuthenticator.swift

```diff
@@ -53,4 +69,127 @@ class SocialMediaAuthenticator: NSObject, GIDSignInDelegate {
             completion?(.success(()))
         }
     }
+    
+    //MARK: - Facebook
+    
+    private func facebookSignIn(sender: UIViewController, completion: @escaping SignInCallback) {
+        let permissions = ["public_profile", "email"]
+        LoginManager().logIn(permissions: permissions, from: sender) { (logInResuts, error) in
+            if let error = error {
+                completion(.failure(error))
+            }
+            else {
+                guard let logInResuts = logInResuts else {
+                    let error = LoginError.noLogInResultsFound
+                    completion(.failure(error))
+                    return
+                }
+                
+                if logInResuts.isCancelled {
+                    let error = LoginError.userCanceledLogIn
+                    completion(.failure(error))
```

Comment:
You don't have to change it, but if instead of creating the error variables you pass the error directly to the completion you will save a couple of lines.